src/Main.js

```diff
@@ -0,0 +1,14 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import App from "./pages/App";
+import FolderPage from "./pages/FolderPage";
+
+export default function Main() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<App />} />
```

Comment:
라우터를 도입하면서 App보다는 어떤 페이지인지 표현되도록 이름이 수정되면 좋을 것 같아요~
```suggestion
        <Route path="/" element={<Home />} />
```

Reply:
요구사항을 확인해보니까 / 경로가 /shared 경로가 되어야 할 것 같네요~
그러면 App도 SharedPage가 되어야 할 것 같구요~